lib/fluent/env.rb

```diff
@@ -21,6 +21,7 @@
 
 module Fluent
   DEFAULT_CONFIG_PATH = ENV['FLUENT_CONF'] || '/etc/fluent/fluent.conf'
+  DEFAULT_CONFIG_INCLUDE_DIR = ENV["FLUENT_CONF_INCLUDE_DIR"] || '/etc/fluent/conf.d'
```

Comment:
I'm sorry, maybe I'm not understanding something, but could you tell me why this needs to be an environment variable?

Reply:
Mainly introduced customizing for fluent-package windows edition.
On windows /etc/fluent/conf.d is not appropriate.

Reply:
I see! Thanks!